plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/tasks/BaseKtLintCheckTask.kt

```diff
@@ -87,6 +92,9 @@ abstract class BaseKtLintCheckTask @Inject constructor(
         convention("256m")
     }
 
+    private var sourceFiles: ConfigurableFileCollection = objectFactory.fileCollection()
+    private val patternSet: PatternFilterable = PatternSet()
```

Comment:
Can an instance be injected? I don't think the constructor will be change, but if it is, this way we're forwards compatible.

Reply:
Unfortunately `PatternFilterable` is not injectable by Gradle, so I changed the code to pass `PatternSet` as a constructor parameter.